configuration/defaults.go

```diff
@@ -113,4 +113,11 @@ vwIDAQAB
 
 	// The number of hours to retain expired tokens.  After this time limit has been exceeded, the token may be cleaned up (deleted)
 	defaultExpiredTokenRetentionHours = 24
+
+	// defaultUserDeactivationFetchLimit the default number of accounts to return when fetching the users to deactivate (and deactivate at once)
+	defaultUserDeactivationFetchLimit = 10
+	// defaultUserDeactivationInactivityPeriod the default number of days of inactivity before a user account can be deactivated
+	defaultUserDeactivationInactivityNotificationPeriod = 30 // 30 days
```

Comment:
```suggestion
	defaultUserDeactivationInactivityNotificationPeriod = 24 // 24 days
```
According to https://docs.google.com/document/d/1fQYxciRt2sYFrTpNunEqOobic1L3swbaPG_ztGKuNt4/edit?disco=AAAACYMKYNI we have a new numbers for notification/deactivation - 24/31 days.

Reply:
done in a3795b4